src/import-utils.ts

```diff
@@ -67,17 +67,18 @@ export async function inlineHtmlImport(
     return;
   }
 
-  // If we've previously visited a url that is part of a bundle, it means we've
-  // handled that entire bundle, so we guard against inlining any other file
-  // from that bundle by checking the visited urls for the bundle url itself.
-  if (visitedUrls.has(importBundleUrl)) {
-    astUtils.removeElementAndNewline(htmlImport);
-    return;
-  }
-
```

Comment:
So I basically MOVED this block into the if below it, because it is only relevant when the importBundleUrl is not the CURRENT bundle url.